core/src/main/java/io/aiven/klaw/model/enums/ResourceType.java

```diff
@@ -0,0 +1,30 @@
+package io.aiven.klaw.model.enums;
+
+import lombok.extern.slf4j.Slf4j;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+@Slf4j
+public enum ResourceType {
```

Comment:
May be we can reuse `RequestEntityType` enum ?

Reply:
I missed that yes we can definitely do that